{"info": {"author": "", "author_email": "", "bugtrack_url": null, "classifiers": ["Framework :: Django", "License :: Public Domain", "Operating System :: OS Independent", "Programming Language :: Python", "Programming Language :: Python :: 3", "Programming Language :: Python :: 3.5", "Programming Language :: Python :: 3.6", "Programming Language :: Python :: 3.7"], "description": "<!--\nhttps://pypi.org/project/readme-generator/\nhttps://pypi.org/project/python-readme-generator/\nhttps://pypi.org/project/django-readme-generator/\n-->\n\n[![](https://img.shields.io/pypi/pyversions/django-traceback.svg?longCache=True)](https://pypi.org/project/django-traceback/)\n\n#### Installation\n```bash\n$ [sudo] pip install django-traceback\n```\n\n#### Models\nmodel|`__doc__`\n-|-\n`Traceback` |Traceback(id, type, value, traceback, path, created_at)\n\n#### Functions\nfunction|`__doc__`\n-|-\n`django_traceback.utils.save_traceback(path=None)` |save a traceback\n\n#### Examples\n```python\nfrom django_traceback.utils import save_traceback\nimport requests\nfrom apps.celery import celery_app\n\n@celery_app.task\ndef task():\n    try:\n        r = requests.get('url')\n    except (requests.exceptions.ConnectionError,...):\n        save_traceback(__file__)\n        # init task again\n```\n\n<p align=\"center\">\n    <a href=\"https://pypi.org/project/django-readme-generator/\">django-readme-generator</a>\n</p>", "description_content_type": "text/markdown", "docs_url": null, "download_url": "", "downloads": {"last_day": -1, "last_month": -1, "last_week": -1}, "home_page": "https://github.com/andrewp-as-is/django-traceback.py", "keywords": "django traceback", "license": "", "maintainer": "", "maintainer_email": "", "name": "django-traceback", "package_url": "https://pypi.org/project/django-traceback/", "platform": "", "project_url": "https://pypi.org/project/django-traceback/", "project_urls": {"Homepage": "https://github.com/andrewp-as-is/django-traceback.py"}, "release_url": "https://pypi.org/project/django-traceback/2019.10.2/", "requires_dist": null, "requires_python": "", "summary": "Traceback model and utils", "version": "2019.10.2", "yanked": false, "html_description": "<div class=\"project-description\">\n            <p><a href=\"https://pypi.org/project/django-traceback/\" rel=\"nofollow\"><img alt=\"\" src=\"https://warehouse-camo.ingress.cmh1.psfhosted.org/b2f354415565af596ff7fed366515a1786055096/68747470733a2f2f696d672e736869656c64732e696f2f707970692f707976657273696f6e732f646a616e676f2d74726163656261636b2e7376673f6c6f6e6743616368653d54727565\"></a></p>\n<h4>Installation</h4>\n<pre>$ <span class=\"o\">[</span>sudo<span class=\"o\">]</span> pip install django-traceback\n</pre>\n<h4>Models</h4>\n<table>\n<thead>\n<tr>\n<th>model</th>\n<th><code>__doc__</code></th>\n</tr>\n</thead>\n<tbody>\n<tr>\n<td><code>Traceback</code></td>\n<td>Traceback(id, type, value, traceback, path, created_at)</td>\n</tr></tbody></table>\n<h4>Functions</h4>\n<table>\n<thead>\n<tr>\n<th>function</th>\n<th><code>__doc__</code></th>\n</tr>\n</thead>\n<tbody>\n<tr>\n<td><code>django_traceback.utils.save_traceback(path=None)</code></td>\n<td>save a traceback</td>\n</tr></tbody></table>\n<h4>Examples</h4>\n<pre><span class=\"kn\">from</span> <span class=\"nn\">django_traceback.utils</span> <span class=\"kn\">import</span> <span class=\"n\">save_traceback</span>\n<span class=\"kn\">import</span> <span class=\"nn\">requests</span>\n<span class=\"kn\">from</span> <span class=\"nn\">apps.celery</span> <span class=\"kn\">import</span> <span class=\"n\">celery_app</span>\n\n<span class=\"nd\">@celery_app</span><span class=\"o\">.</span><span class=\"n\">task</span>\n<span class=\"k\">def</span> <span class=\"nf\">task</span><span class=\"p\">():</span>\n    <span class=\"k\">try</span><span class=\"p\">:</span>\n        <span class=\"n\">r</span> <span class=\"o\">=</span> <span class=\"n\">requests</span><span class=\"o\">.</span><span class=\"n\">get</span><span class=\"p\">(</span><span class=\"s1\">'url'</span><span class=\"p\">)</span>\n    <span class=\"k\">except</span> <span class=\"p\">(</span><span class=\"n\">requests</span><span class=\"o\">.</span><span class=\"n\">exceptions</span><span class=\"o\">.</span><span class=\"n\">ConnectionError</span><span class=\"p\">,</span><span class=\"o\">...</span><span class=\"p\">):</span>\n        <span class=\"n\">save_traceback</span><span class=\"p\">(</span><span class=\"vm\">__file__</span><span class=\"p\">)</span>\n        <span class=\"c1\"># init task again</span>\n</pre>\n<p align=\"center\">\n    <a href=\"https://pypi.org/project/django-readme-generator/\" rel=\"nofollow\">django-readme-generator</a>\n</p>\n\n          </div>"}, "last_serial": 5920452, "releases": {"2019.10.2": [{"comment_text": "", "digests": {"md5": "9104711801a70ed5d15f69ea74d90e63", "sha256": "4b48669cdd35fb7cfff068016c366ac4c47cb6e13715c069cd078bb54cfeb87c"}, "downloads": -1, "filename": "django-traceback-2019.10.2.tar.gz", "has_sig": false, "md5_digest": "9104711801a70ed5d15f69ea74d90e63", "packagetype": "sdist", "python_version": "source", "requires_python": null, "size": 2302, "upload_time": "2019-10-02T20:51:27", "upload_time_iso_8601": "2019-10-02T20:51:27.774882Z", "url": "https://files.pythonhosted.org/packages/ee/57/81677596ea5731a70fcc08e1804d6db61cf98e49c4ce9f35d8b9d986b1b8/django-traceback-2019.10.2.tar.gz", "yanked": false}], "2019.4.13": [{"comment_text": "", "digests": {"md5": "235ed20670d70cfe7b4efe0915e50c32", "sha256": "be4500c7ea7420e3fc63a3920ac652fb952cb7f0ae932a9923385f402ec3e1c8"}, "downloads": -1, "filename": "django-traceback-2019.4.13.tar.gz", "has_sig": false, "md5_digest": "235ed20670d70cfe7b4efe0915e50c32", "packagetype": "sdist", "python_version": "source", "requires_python": null, "size": 2301, "upload_time": "2019-10-01T12:48:34", "upload_time_iso_8601": "2019-10-01T12:48:34.251718Z", "url": "https://files.pythonhosted.org/packages/b4/50/cfbe4dfdc4f5dc31bab7fa2da5390a680d9c02927b34d1119db8412c812b/django-traceback-2019.4.13.tar.gz", "yanked": false}]}, "urls": [{"comment_text": "", "digests": {"md5": "9104711801a70ed5d15f69ea74d90e63", "sha256": "4b48669cdd35fb7cfff068016c366ac4c47cb6e13715c069cd078bb54cfeb87c"}, "downloads": -1, "filename": "django-traceback-2019.10.2.tar.gz", "has_sig": false, "md5_digest": "9104711801a70ed5d15f69ea74d90e63", "packagetype": "sdist", "python_version": "source", "requires_python": null, "size": 2302, "upload_time": "2019-10-02T20:51:27", "upload_time_iso_8601": "2019-10-02T20:51:27.774882Z", "url": "https://files.pythonhosted.org/packages/ee/57/81677596ea5731a70fcc08e1804d6db61cf98e49c4ce9f35d8b9d986b1b8/django-traceback-2019.10.2.tar.gz", "yanked": false}], "timestamp": "Fri May  8 00:52:23 2020"}